example/flaskr/templates/recipe/index.html

```diff
@@ -1,28 +1,51 @@
 {% extends 'base.html' %}
 
 {% block header %}
-<h1>{% block title %}Recipes{% endblock %}</h1>
-
-<a class="action" href="{{ url_for('recipe.create') }}">New</a>
-
+  <h1>{% block title %}Recipes{% endblock %}</h1>
+  <a class="action" href="{{ url_for('recipe.create') }}">New Recipe</a>
 {% endblock %}
 
 {% block content %}
-{% for recipe in recipes %}
-<article class="post">
-    <header>
+<form method="POST">
+  <label for="search">Search</label>
+  <input name="search" id="search" value="{{ request.form['search'] }}">
+  <label for="filter">Filter</label>
+  <div>
+  <input type = "radio"
+        name = "filter"
+        id = "title"
+        value = "title"
+        checked = "checked" />
+        <label for="title">By title</label>
+  <input type = "radio"
+      name = "filter"
+      id = "category"
+      value = "category" />
+      <label for="category">By category</label>
+  <input type = "radio"
+        name = "filter"
+        id = "username"
+        value = "username"/>
+        <label for="username">By author</label>
+  </div>
+  <input type="submit" value="Search">
+</form>
+  {% for recipe in recipes %}
+    <article class="recipe">
+      <header>
         <div>
-            <h1>{{ recipe['title'] }}</h1>
-            <div class="about">by {{ recipe['username'] }} on {{ recipe['created'].strftime('%Y-%m-%d') }}</div>
+          <h1>{{ recipe['title'] }}</h1>
+          <div class="about">by {{ recipe['username'] }} on {{ recipe['created'].strftime('%Y-%m-%d') }}</div>
         </div>
-
```

Comment:
This is the only thing I would leave actually. The select will bring us to a view template where edit and save to recipe book should be there.  In F1-01, one requirement is to be able to view recipe

Reply:
AH, good catch. I actually think this got cleaned up when I was dealing with merge conflicts. I'll fix it up

Reply:
Just committed, should have fixed it :+1: